src/docs/sdk/sessions.mdx

```diff
@@ -321,6 +392,19 @@ In either case, the `init` flag must be set correctly for the _first
 transmission_ of the session, and session metadata such as the distinct ID must
 be immutable after the initial transmission.
 
+### Pre-aggregation of sessions
+
+If an SDK is configured to use _server-mode_ sessions, it should group and
+pre-aggregate session counts before sending them to sentry. Whenever a session
+is being closed (transitions to a terminal state), and it was not previously
+being sent upstream (its `init` flag would be `true`), it is eligible for
+aggregation. The `started` timestamp of the session should be rounded down to
+minutes, and it must be aggregated into the bucket identified by that rounded
+timestamp, and the sessions distinct id (`did`). In the appropriate bucket,
+increase the session count corresponding to the sessions status. Contrary to
+individual session updates, the `"errored"` state is used to mark sessions that
+have the `"exited"` state, and a non-zero `errors` count.
```

Comment:
This description is super useful. Essentially, it lists a few conditions to check and steps to execute for aggregating sessions. I was wondering if this could be formatted as a list for easier comprehension.